### PR TITLE
Add Standalone builds

### PR DESCRIPTION
**Pull request by nikitalita**
This adds standalone binary builds to the CI for each platform (macos, linux, windows).

This was a accomplished by creating a new darknet binding library for auto-builds (https://github.com/nikitalita/libdarknetpy) and uploading the word bubble detector to a seperate repo to download (https://github.com/nikitalita/Bubble-detection-model)

This also adds a better box sorter that more effectively sorts boxes according to their guessed order.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[project]
+name = "ocr_aggregator_server"
+authors = [
+    {name = "millenniumbug", email = "millenniumbug@gmail.com"},
+]
+description = "OCR Aggretgator server for Magic Translatior"
+requires-python = ">=3.7"
+license = {text = "MIT"}
+classifiers = [
+    "Framework :: Django",
```
Comment (milleniumbug) [inline pyproject.toml]: I... don't think I'm using Django here?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Utilities
+
+[options]
+install_requires =
+    pytest>=3.0.7
+    Flask>=2.0.2
+    flask-cors>=3.0.10
+    manga-ocr>=0.1.11
+    fugashi>=1.3.0
```
Comment (milleniumbug) [inline setup.cfg]: What do we use `fugashi` for?

Comment (nikitalita) [reply setup.cfg]: manga-ocr uses it; I'm pinning it at >=1.3.0 to make sure we get the version with my macos fixes.

File: pyproject.toml (hunk 1 above)
Comment (nikitalita) [reply pyproject.toml]: all these files should probably be cleaned up...

**Pull request merged**